template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/JwtBearerOptionsSetup.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.AspNetCore.Authentication.JwtBearer; 
+using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Options;
 using System;
```

Comment:
Removing from this PR. Handle as part of updating the web api template: https://github.com/dotnet/templating/pull/700